src/doc/unstable-book/src/compiler-environment-variables/SDKROOT.md

```diff
@@ -0,0 +1,6 @@
+# `SDKROOT`
+
+This environment variable is used on MacOS targets.
+It is passed through to the linker (either as `-isysroot` or `-syslibroot`) without changes.
```

Comment:
I still plan to change this in https://github.com/rust-lang/rust/pull/131477 (though if you want, you can keep the text as-is for now).

```suggestion
It is passed through to the linker (currently either as `-isysroot` or `-syslibroot`).
```